tests/config/Shared/stores.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+$stores = [];
+
+$stores['DE'] = [
+    'queuePools' => [
+        'synchronizationPool' => [
+            'DE-connection',
+        ],
+    ],
+    'locales' => [
+        'de' => 'de_DE',
+    ],
+    'countries' => [
+        'DE',
+        'AT',
+        'NO',
+        'CH',
+        'ES',
+        'GB',
+    ],
+];
+$stores['AT']['queuePools']['synchronizationPool'] = ['DE-connection'];
+$stores['US']['queuePools']['synchronizationPool'] = ['DE-connection'];
+
+return $stores;
```

Comment:
No need for currency, etc.? Imho you can just copy config/Shared/stores.php from project lvl